internal/decryptor/decryptor.go

```diff
@@ -697,9 +697,13 @@ func recurseKustomizationFiles(root, path string, visit visitKustomization, visi
 		return err
 	}
 
+	// Components may contain resources as well, ...
+	// ...so we have to process both .resources and .components values
+	resources := append(kus.Resources, kus.Components...)
```

Comment:
Please write a test for decrypting an env file found in a component, this should be added to https://github.com/fluxcd/kustomize-controller/blob/main/internal/controller/kustomization_decryptor_test.go

Reply:
@stefanprodan please, check the changes